src/ploneintranet/workspace/browser/tiles/templates/sidebar-settings-security.pt

```diff
@@ -0,0 +1,81 @@
+	                    <strong>Self-managed:</strong> Outsiders can self-join becoming a workspace member.
+	                </p>
+	                <label>
+	                    Particpant policy
+	                    <input name="partpol" type="range" step="1" min="1" max="4" value="2" /> 
+	                    <span class="four slider-labels">
+	                        <span class="label pat-depends" data-pat-depends="condition: partpol=1; action: enable">Consume&nbsp;<a href="#help-partpol-1" class="iconified icon-info-circle pat-tooltip" data-pat-tooltip="source: ajax;">Info</a></span>
+	                        <span class="label pat-depends" data-pat-depends="condition: partpol=2; action: enable">Produce&nbsp;<a href="#help-partpol-2" class="iconified icon-info-circle pat-tooltip" data-pat-tooltip="source: ajax;">Info</a></span>
+	                        <span class="label pat-depends" data-pat-depends="condition: partpol=3; action: enable">Publish&nbsp;<a href="#help-partpol-3" class="iconified icon-info-circle pat-tooltip" data-pat-tooltip="source: ajax;">Info</a></span>
+	                        <span class="label pat-depends" data-pat-depends="condition: partpol=4; action: enable">Moderate&nbsp;<a href="#help-partpol-4" class="iconified icon-info-circle pat-tooltip" data-pat-tooltip="source: ajax;">Info</a></span>
+
+	                    </span>
+	                </label>
+	                <p id="help-partpol-1" class="pat-message notice pat-depends" data-pat-depends="condition: partpol=1">
+	                    <strong>Consume:</strong> Workspace members can read content. They cannot add, publish or edit content.
+	                </p>
+	                <p id="help-partpol-2" class="pat-message notice pat-depends" data-pat-depends="condition: partpol=2">
+	                    <strong>Produce:</strong> Workspace members can read and add content. They can neither publish nor edit content.
+	                </p>
+	                <p id="help-partpol-3" class="pat-message notice pat-depends" data-pat-depends="condition: partpol=3">
+	                    <strong>Publish:</strong> Workspace members can read, add and publish content. They cannot edit other member's content.
+	                </p>
+	                <p id="help-partpol-4" class="pat-message notice pat-depends" data-pat-depends="condition: partpol=4">
+	                    <strong>Moderate:</strong> Workspace members can do everything: read, add, publish and edit content.
+	                </p>
+	                <p>
+	                    The participant policy is the default permission settings for all members of this workspace. Execptions for individual people may be made on the <a href="/feedback/workspace-settings-members.html" class="pat-inject" data-pat-inject="target: #workspace-settings; source: #workspace-settings">Members tab</a>.
+	                </p>
+	            </fieldset>
+		    </form>
+		</div>
+	</div>
+</div>
+</body>
+</html>
```

Comment:
I'm no Jenkins expert but could we do something along these lines:
http://www.phase2technology.com/blog/having-the-butler-tidy-up-your-html/